capstone/src/main/java/com/google/sps/servlets/auth/AuthServlet.java

```diff
@@ -19,12 +19,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     UserService userService = UserServiceFactory.getUserService();
     if (userService.isUserLoggedIn()) {
       String logoutUrl = userService.createLogoutURL(redirectURL);
-      response
-          .getWriter()
-          .println(
-              "<button class='auth-button' onclick='window.location.href=\""
-                  + logoutUrl
-                  + "\"'>Logout Here</button>");
+      response.getWriter().println("<a href=\"" + logoutUrl + "\">Logout</a>");
```

Comment:
For oneliners like this it's probably not worth changing, but for bigger pieces of code that needs to be loaded into HTML, maybe you can use a template so you can just return the URL from the servlet and load the url into the href for the desired element.

Reply:
Yeah, I think it might not work for this case (or it might be messier) since it's just one line but I did move the one-liner code to JS. I should be able to switch the inbox and club list to templates so I'll go ahead and do that. Thanks for bringing this up!